calabash/Classes/FranklyServer/Routes/LPBackdoorRoute.m

```diff
@@ -29,40 +34,41 @@ - (NSDictionary *) JSONResponseForMethod:(NSString *) method URI:(NSString *) pa
     selectorName = [selectorName stringByAppendingString:@":"];
   }
 
+  id argument = [data objectForKey:@"arg"];
+  if (!argument) {
+    LPLogError(@"Expected data dictionary to contain an 'arg' key");
```

Comment:
re: NSNull - You are correct.  I started working on that, but didn't stress about it because there are other things to focus on.  

@re:  `__self__` - https://github.com/calabash/calabash-ios-server/blob/develop/XCTest/Tests/Invoker/LPInvokerSelectorWithArgumentsTest.m#L348

We could!  Pretty good idea!